h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoostModel.java

```diff
@@ -35,7 +35,7 @@
         implements SharedTreeGraphConverter, Model.LeafNodeAssignment, Model.Contributions {
 
   private static final String PROP_VERBOSITY = H2O.OptArgs.SYSTEM_PROP_PREFIX + ".xgboost.verbosity";
-  private static final String PROP_NTHREAD = SYSTEM_PROP_PREFIX + "xgboost.nthread";
+  private static final String PROP_NTHREAD = SYSTEM_PROP_PREFIX + "xgboost.nthreadMax";
```

Comment:
This appears to require a change in documentation, yet there is no JIRA tied with this PR.

Reply:
the property is in experimental mode and has not been documented yet